index.html

```diff
@@ -2,19 +2,22 @@
 <head>
   <title>My First Chatbot</title>
   <link rel="stylesheet" href="style.css">
+  <script src="chatbot.js" defer></script>
+
 </head>
 
 <body>
   <div id="demo">
       <h1>My first chatbot!</h1>
       <h3>Talk to your bot!</h3>
-      <input id="input" type="text" placeholder="Say something" />
+      <input id="oranges" type="text" placeholder="Say something" />
```

Comment:
Was this line just for testing? 
Anyway, it's a good idea to give your ids meaningful names :smile: